content/ember-released-4-11.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Ember 4.11 Released
+authors:
+  - jen-weber
+date: 2023-02-25T12:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.11 of Ember.js, Ember Data, and Ember CLI. <!-- Block start: Uncomment if an LTS candidate --><!--This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.--><!-- Block end -->
+
+This release kicks off the 4.11.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 4.11
+
+Ember.js 4.11 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 4.11 introduced 0 bug fixes.
+
+#### Features
+
+Ember.js 4.11 introduced 2 features.
+
+1. Feature description
+2. Feature description
```

Comment:
````suggestion
1. Stable TypeScript types for the `@ember/owner` package (first released in 4.10).
2. Stable TypeScript types for the `@ember/error` package.

Both of these packages already had tyeps available as part of [the public preview types](https://blog.emberjs.com/announcing-official-typescript-types-public-preview/), so you might wonder: _What is new here?_ The answer is: these are the first type definitions generated from Ember's own TypeScript source code, instead of written by hand. This means that these type are _guaranteed_ to match Ember's own behavior: they literally cannot get out of sync.

If you are already using the public preview types, as documented in [the announcement post](https://blog.emberjs.com/announcing-official-typescript-types-public-preview/), you don't have to do anything. If you would like to opt into using the new stable and preview types, you can add this to your project (e.g. in `app.ts`):

```ts
import 'ember-source/types';
import 'ember-source/types/preview';
```

**Reminder:** while these two packages are now stable, much of Ember's types are still in preview, and subject to change between minor releases. We expect to finish getting the rest of Ember's types stable, and therefore following Ember's normal stability rules, early in the upcoming 5.x releases!
````